forest/src/cli/chain_cmd.rs

```diff
@@ -32,9 +38,9 @@ pub enum ChainCommands {
         /// Skip old messages
         #[structopt(short)]
         skip_old_messages: bool,
-        /// Snapshot output path. Default to `forest_snapshot_{year}-{month}-{day}_height_{height}.car`
-        #[structopt(short)]
-        output_path: Option<String>,
+        /// Snapshot output path. Default to `forest_snapshot_{chain_}{year}-{month}-{day}_height_{height}.car`
+        #[structopt(short, default_value = OUTPUT_PATH_DEFAULT_VALUE, help = "format in of the filename of the snapshot")]
```

Comment:
Use the doc comment as the help string.

Reply:
`StructOpt` can use the doc-comment for the help string. So, you can remove the `help = "..."` part.

Reply:
Fixed